mailstore-connector/src/com/axonivy/connector/mailstore/MailStoreService.java

```diff
@@ -82,7 +88,24 @@ public static MessageIterator messageIterator(String storeName, String srcFolder
 	 */
 	public static MessageIterator messageIterator(String storeName, String srcFolderName, String dstFolderName,
 			boolean delete, Predicate<Message> filter, Comparator<Message> comparator) {
-		return new MessageIterator(storeName, srcFolderName, dstFolderName, delete, filter, comparator);
+		return new MessageIterator(storeName, srcFolderName, Arrays.asList(dstFolderName), delete, filter, comparator);
+	}
+	
+	/**
+	 * Get a {@link MessageIterator}.
+	 * 
+	 * @param storeName     name of Email Store (Imap Configuration)
+	 * @param srcFolderName source folder name
+	 * @param dstFolderNames list destination folder will be moved to these folder
+	 * @param delete        delete mail from source folder?
+	 * @param filter        a filter predicate
+	 * @param sort          a sort comparator
+	 * @return
+	 * @throws MessagingException
+	 */
+	public static MessageIterator messageIterator(String storeName, String srcFolderName, 
+			boolean delete, Predicate<Message> filter, Comparator<Message> comparator, List<String> dstFolderNames) {

```

Comment:
Can we use `String...` instead? So that for current usages, they don't need to change anything. The method can be called:
-  new messageIterator(storeName, srcFolderName, delete, filter, comparator, dstFolderName) - `dstFolderName` can be a single string
-   new messageIterator(storeName, srcFolderName, delete, filter, comparator, dstFolderNames) - `dstFolderNames` can be an array of String